OWNERS

```diff
@@ -4,6 +4,9 @@ reviewers:
   - sttts
   - mfojtik
   - soltysh
+  - adambkaplan
+  - coreydaley
```

Comment:
I don't know if I'm ready for this kind of responsibility ...

Reply:
interestingly the bot let you lgtm https://github.com/openshift/cluster-openshift-controller-manager-operator/pull/59 anyway, so apparently this doesn't even matter....